Content.Shared/Health/BodySystem/Mechanisms/Mechanism.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System;
+using System.Collections.Generic;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Serialization;
+using Robust.Shared.ViewVariables;
+using YamlDotNet.RepresentationModel;
+
+
+
+
+namespace Content.Shared.BodySystem {
+
+    /// <summary>
+    ///     Data class representing a persistent item inside a BodyPart. This includes livers, eyes, cameras, brains, explosive implants, binary communicators, etc.
+    /// </summary>
+    [NetSerializable, Serializable]
+    public class Mechanism {
+
+        [ViewVariables]
+        public string Name;
+		
+		/// <summary>
+		///     Description shown in a mechanism installation console or when examining an uninstalled mechanism.
+		/// </summary>				
+		[ViewVariables]
+        public string Description;		
+		
+		/// <summary>
+		///     The message to display upon examining a mob with this mechanism installed. If the string is empty (""), no message will be displayed.  
+		/// </summary>			  
+		[ViewVariables]
+		public string ExamineMessage;
+
+        /// <summary>
+        ///     Path to the .png that represents this mechanism (NOT .rsi folder).
```

Comment:
Why?

Reply:
I dunno. Should it be an .rsi? SpriteComponents can either be a Texture .png path or a .rsi folder path + a state, not both. I just chose the former since it's easier.

Reply:
I feel it should be an rsi, yeah.
Dunno others opinions on that. (ask PJB)